tests/Unit/Domain/Structure/Test_ExcisionSphere.cpp

```diff
@@ -101,8 +101,8 @@ void test_abutting_direction_shell() {
         true,
         std::nullopt,
         {2.},
-        {domain::CoordinateMaps::Distribution::Linear,
-         domain::CoordinateMaps::Distribution::Linear}};
```

Comment:
(nitpick) you can remove the outer braces

Reply:
Decided to leave this as is. Personally I like seeing the outer braces because then I know this function doesn't actually take a vector as an argument; it takes something else (even if that something is constructible from just the vector)

Reply:
Ok